app.js

```diff
@@ -0,0 +1,450 @@
+    },
+    
+    createProfileContainer(className) {
+        const article = document.createElement('article');
+        article.classList.add(className);
+        return article;
+    },
+    
+    createProfileHeading(person, className) {
+        const heading = document.createElement('h2');
+        heading.textContent = person.name;
+        heading.classList.add(className);
+        return heading;
+    },
+    
+    createProfileStartDate(person, className) {
+        const label = document.createElement('label');
+        label.textContent = person.date;
+        label.classList.add(className);
+        return label;
+    },
+    
+    createProfileGroup(person, className) {
+        const label = document.createElement('label');
+        label.textContent = person.group;
+        label.classList.add(className);
+        return label;
+    },
+    
+    createProfileRecommendation(person, className) {
+        const label = document.createElement('label');
+        label.textContent = person.recommendedBy;
+        label.classList.add(className);
+        return label;
+    },
+        
+    createProfile() {
+        const list = app.listOfPeople;
+        list.forEach(person => {
+            const article = this.createProfileContainer('person-shower');
+            const heading = this.createProfileHeading(person, 'name-shower');
+            const startDate = this.createProfileStartDate(person, 'start-date-shower');
+            const group = this.createProfileGroup(person, 'group-shower');
+            const recommendation = this.createProfileRecommendation(person, 'recommendation-shower');
+            article.appendChild(heading);
+            article.appendChild(startDate);
+            article.appendChild(group);
+            article.appendChild(recommendation);
+            const profile = document.querySelector('.profile');
+            profile.appendChild(article);
+        });
+    },
+    
+    createCheckbox(date) {
+        const checkbox = document.createElement('input');
+        checkbox.setAttribute('type', 'checkbox');
+        checkbox.setAttribute('data-text', date);
+        checkbox.setAttribute('onchange', 'handlers.addPersonToAttendenceList(event)');
+        return checkbox;
+    },
+      
+    addPersonToAttendenceList(event) {
+        const nameOfThePerson = document.querySelector('.person-name').textContent;
+        const date = event.target.dataset.text;
+        app.attendanceList.push({date, people: [] });
+        appStorage.setAttendanceList();
+    },
+    
+    displayAttendenceList() {
+        const table = document.querySelector('.attendence-people-list');
+        table.innerHTML = '';
+        const list = app.listOfPeople;
+        const saturdaysDate = document.querySelector('.saturday-display').textContent;
+        const sundaysDate = document.querySelector('.sunday-display').textContent;
+        list.forEach(person => {
+            const tr = this.createTrElement();
```

Comment:
extract the `.forEach()` callback into a separate method to make `displayAttendanceList` more readable